typings/index.d.ts

```diff
@@ -1889,6 +1889,9 @@ declare module 'discord.js' {
 		readonly client: Client;
 		id: Snowflake;
 		token: string;
+		readonly createdAt: Date;
+		readonly createdTimestamp: number;
+		readonly url: String;
```

Comment:
```suggestion
		readonly url: string;
```

This should use the primitive `string` type, not the `String` global type.